mavros_extras/src/plugins/fake_gps.cpp

```diff
@@ -227,7 +227,21 @@ class FakeGPSPlugin : public plugin::PluginBase,
 			ROS_INFO_STREAM("FGPS: Caught exception: " << e.what() << std::endl);
 		}
 
-		Eigen::Vector3d vel = (old_ecef - current_ecef) / (stamp.toSec() - old_stamp);	// [m/s]
+		Eigen::Matrix<double, 3, 3> dcm_ned_ecef;
+		double lat = geodetic.x();
+		double lon = geodetic.y();
+		dcm_ned_ecef(0, 0) = -sin(lat)*cos(lon);
+		dcm_ned_ecef(0, 1) = -sin(lat)*sin(lon);
+		dcm_ned_ecef(0, 2) = cos(lat);
+		dcm_ned_ecef(1, 0) = -sin(lon);
+		dcm_ned_ecef(1, 1) = cos(lon);
+		dcm_ned_ecef(1, 2) = 0;
+		dcm_ned_ecef(2, 0) = -cos(lat)*cos(lon);
+		dcm_ned_ecef(2, 1) = -cos(lat)*sin(lon);
+		dcm_ned_ecef(2, 2) = -sin(lat);
```

Comment:
Perhaps `<<` or initializer list would be more readable:

```
dcm_ned_ecef {
 {1, 2, 3},
 {4, 5, 6},
 {7, 8, 9},
};
```
https://eigen.tuxfamily.org/dox/group__TutorialMatrixClass.html

Reply:
Thanks for the review. I couldn't get brace initializer lists to work with eigen, so went with the stream op approach.